libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -195,20 +195,20 @@ void YamlConfiguration::parseProcessorNodeYaml(YAML::Node processorsNode, core::
 
         if (procCfg.schedulingStrategy == "TIMER_DRIVEN" || procCfg.schedulingStrategy == "EVENT_DRIVEN") {
           if (core::Property::StringToTime(procCfg.schedulingPeriod, schedulingPeriod, unit) && core::Property::ConvertTimeUnitToNS(schedulingPeriod, unit, schedulingPeriod)) {
-            logger_->log_debug("convert: parseProcessorNode: schedulingPeriod => [%ll] ns", schedulingPeriod);
+            logger_->log_debug("convert: parseProcessorNode: schedulingPeriod => [%lld] ns", schedulingPeriod);
```

Comment:
Use `PRId64` from `<cinttypes>` for `int64_t`. `lld` is meant to be used for `long long int`, which is not guaranteed to be the same as `int64_t`. It is not the same on my platform.

The above applies to all 7 changes in this file.

Reply:
fixed by da22085